packages/core-components/src/layout/Sidebar/SidebarSubmenuItem.tsx

```diff
@@ -185,6 +185,10 @@ export const SidebarSubmenuItem = (props: SidebarSubmenuItemProps) => {
               classes.item,
               isActive ? classes.selected : undefined,
             )}
+            style={{
+              textTransform: 'none',
+              justifyContent: 'flex-start',
+            }}
```

Comment:
Hm. We avoid using `style`, and instead prefer class names in the `makeStyles` above. Mixing these gets really iffy.

Reply:
Thanks for info! Moved it to makeStyles now.